### ISSUE TITLE
Download button display in search results

### ISSUE THREAD

**Issue by corylown** [duke-libraries/ddr-public#417]
For digital collections items that display a download button on the show page there isn't a corresponding download button for these items on their results list display.

<img width="930" alt="screen shot 2016-06-03 at 7 59 56 am" src="https://cloud.githubusercontent.com/assets/458247/15777990/550d5508-2961-11e6-9a6d-285cb1f0b497.png">

For non-dc collections there is a download link (on components).

<img width="875" alt="screen shot 2016-06-03 at 7 56 29 am" src="https://cloud.githubusercontent.com/assets/458247/15777988/5176f4ee-2961-11e6-9c24-678299d0c269.png">

We may want to be more deliberate about when and where the download button appears in the interface.

**Comment by seanaery** [comment]
Good point. This will be a bit complicated with Digital Collections since in that case_usually_ (but not always) the file a user wants to and is authorized to download is the derivative, not the actual TIF, MOV, WAV, etc. file stored in the repository.
